types/applepayjs/index.d.ts

```diff
@@ -773,6 +792,11 @@ declare namespace ApplePayJS {
          * An array of payment token contexts that requests multiple payment tokens to support a multimerchant payment.
          */
         multiTokenContexts?: ApplePayPaymentTokenContext[];
+
+        /**
+         * Optional user-defined data.
```

Comment:
```suggestion
         * A Boolean value that determines whether the payment sheet displays the coupon code field.
```

Reply:
Done, thanks!
Could you please take a look?